envs/monkey_zoo/blackbox/config_templates/performance.py

```diff
@@ -58,5 +60,7 @@ class Performance(ConfigTemplate):
             "10.2.2.23",
             "10.2.2.24",
             "10.2.2.25",
+            "10.2.2.45",
```

Comment:
I think these two machines should be on the same subnet to avoid confusion.

Reply:
I agree on that. Presumably `powershell-45` to be moved into `europe-west1-b` ?

Reply:
Yep.